core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/IssueDownloader.java

```diff
@@ -66,17 +67,19 @@ public IssueDownloader(IssueApi issueApi, SourceApi sourceApi, IssueStorePaths i
    * If the component doesn't exist or it exists but has no issues, an empty iterator is returned.
    *
    * @param key project key, or file key.
+   * @param branchName name of the branch. If null - issues will be downloaded for all branches.
```

Comment:
This is not how I understand the source code that handles `batch/issues`, I asked the SQ team about it and they confirmed that the default behavior (when `branch` is not set) is to return issues from the main branch only.

Reply:
Thanks. I'll change the javadoc.